src/components/forms/FamilyForm.tsx

```diff
@@ -488,6 +488,36 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
                   />
                 </FormControl>
               )}
+              {loadedFamily && (
```

Comment:
Could we wrap this all in one check for `loadedFamily` ?

Reply:
Updated